packages/website/docs/Api/ControlFlow.mdx

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 11
+title: Control Flow
+---
+
+import { SquiggleEditor } from "../../src/components/SquiggleEditor";
+import Admonition from "@theme/Admonition";
+
+This page documents control flow. Squiggle has with if/else statements. Use reduce/map constructs instead of for loops.
+
+## If-else
+
+### If-else
+
+```js
+if condition then result else alternative
+```
+
+<SquiggleEditor
+  defaultCode={`
+if x == 1 then 1 else 2
```

Comment:
This and following examples are broken because `x` is not defined.

Reply:
Now fixed.